### PR TITLE
upcoming: [M3-7694] - OBJ Multi Cluster Copy updates

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝
Update final copy for OBJ Multi Cluster.

### Prerequisites
(How to setup test environment)
- Turn on MSW 
- Turn on OBJ Multicluster 

### Verification steps 
(How to verify changes)
- Validate the copy updates mentioned in the ticket, referring to the copy changes from Figma linked within the ticket.
- Ensure there no regression in Bucket, Access key Create / Edit flows when the flag and MSW is off. 

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [ ] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

---

### CODE REVIEW

**Comment by jaalah-akamai** [comment]
I'm not sure this is related, but when going to `http://localhost:3000/object-storage/access-keys` without MSW on, the page breaks. Is the same happening for you?

**Comment by cpathipa** [comment]
> I'm not sure this is related, but when going to `http://localhost:3000/object-storage/access-keys` with feature flag on and MSW off, the page breaks. Is the same happening for you?

@jaalah-akamai This is handled in the PR https://github.com/linode/manager/pull/10189 to manage these errors gracefully when the feature flag is on and MSW is off. Once it is merged, we won't see the app crashing when MSW is off.

**Review by bnussman-akamai — APPROVED** [review]

**Review by mjac0bs — APPROVED** [review]
Thanks Chandra! Confirmed copy changes matched the requested updates, but left a bit of feedback. 

This is my first time looking at OBJ Multi-Cluster stuff in a while, so I also wanted to note an unrelated-to-this-PR UI feature that initially confused me:

Screenshot 2024-02-15 at 8 38 17 AM

After creating an access key, when viewing the modal, I thought the "Copy all" button applied to every field (copy the regions, access key, and secret key all at once) because I only had one region and "Copy all" was at the top. After seeing the HostNamesDrawer, I understand that it is intended to copy *only* the hostnames, of which there can be multiple. Not sure if this would be a common misconception, but might be worth discussing with UX, removing the "Copy all" in this modal, or changing it to "Copy" when there is just one region -- in a separate ticket. 

Another minor note here is that the top and bottom padding on this field seem a little too wide.

**Comment by cpathipa** [comment]
Thank you @mjac0bs  Great feedback.

Copy All - 
- Hides Copy all when one hostname exists. ([fad1190](https://github.com/linode/manager/pull/10188/commits/fad119020690e505e5b1a16db647d8fdbcd2a251)) - (UX approval)
- Adjusted the padding for the field (3068f00). 
  image

Error Message - UX is working on it, Will create a followup PR for better messaging.

File: packages/validation/src/objectStorageKeys.schema.ts [hunk 1]
```diff
@@ -22,6 +22,9 @@ export const updateObjectStorageKeysSchema = object({
     .trim(),
   regions: array()
     .of(string())
-    .min(1, 'Regions must include at least one region')
+    .min(
+      1,
+      'You need to select at least one region. To delete all keys, go to the Access Keys page in Cloud Manager and select Revoke.'
+    )
```
[the remark this answers is not in the document]
Comment (cpathipa) [reply packages/validation/src/objectStorageKeys.schema.ts:28]: Good catch, UX is working on it, Will create a followup PR for better messaging.

**Pull request merged**